module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationLogo.md

```diff
@@ -79,10 +83,11 @@ Accept wildcard characters: False
 ```
 
 ### -FilePath
+
 If provided, the application logo is copied with a random filename to the file path that is specified in this parameter.
```

Comment:
If provided, the application logo is saved to the specified file path using a random file name.

Reply:
Suggestion added. Thank you.